diff --git a/node_modules/@docusaurus/core/lib/webpack/base.js b/node_modules/@docusaurus/core/lib/webpack/base.js
index b409829..67384af 100644
--- a/node_modules/@docusaurus/core/lib/webpack/base.js
+++ b/node_modules/@docusaurus/core/lib/webpack/base.js
@@ -84,7 +84,7 @@ function createBaseConfig(props, isServer, minify) {
                 ? [
                     new terser_webpack_plugin_1.default({
                         cache: true,
-                        parallel: true,
+                        parallel: false, // https://github.com/facebook/docusaurus/pull/3497
                         sourceMap: false,
                         terserOptions: {
                             parse: {
